### ISSUE TITLE
Deliver OCP config with created repositories

### ISSUE THREAD

**Issue by michaelsauter** [opendevstack/ods-quickstarters#243]
The templates used to create the resources should be placed in the repo (or the projects `ocp-config` repo) so that developers can continue to evolve their infrastructure with those templates and Tailor.

**Comment by michaelsauter** [comment]
One issue with this is that it would only make sense to do this, when users can evolve the templates and have them applied via Jenkins. But once that is done, they could not use the UI because the templates would revert the changes.

I think I'd target 4.0, as we also have a topic to think about quickstarter parameters - maybe users should be able to pick if they want the templates or not.

**Comment by michaelsauter** [comment]
Superseded by #959.
